bundles/org.eclipse.ui.workbench/.classpath

```diff
@@ -2,7 +2,8 @@
 <classpath>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-17"/>
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
-	<classpathentry kind="src" path="Eclipse UI"/>
+	<classpathentry kind="src" path="eclipseui"/>
+        <classpathentry kind="src" path="Eclipse UI"/>
```

Comment:
Why do you keep "Eclipse UI" if you renamed it to eclipseui?

Reply:
There are more files in Eclipse UI yet to be moved. if I remove it now, the build will break due to missing files in the available classpath. the entry `Eclipse UI` is safe to be removed only after all the bundles are modified and landed.

Reply:
What is a "bundle" for you?

Reply:
Yes, that's it. Sorry for the long and mismatched communication.